sql/5_0_2-to-5_0_3_upgrade.sql

```diff
@@ -439,3 +439,7 @@ ALTER TABLE `form_eye_mag_prefs` MODIFY `ordering` smallint(6) DEFAULT NULL;
 #IfNotColumnType codes code_text_short varchar(255)
 ALTER TABLE `codes` MODIFY `code_text_short` varchar(255) NOT NULL default '';
 #EndIf
+
+#IfNotColumnType amendments created_time timestamp NULL
+ALTER TABLE `amendments` MODIFY `created_time` timestamp NULL COMMENT 'created time';
```

Comment:
hi @ShubhamBansal1997 ,
sadly, on testing, this always gets run (even when the column has already been changed to timestamp NULL), which makes sense since Type and Default are stored in separate columns:
![image](https://user-images.githubusercontent.com/278968/79947009-f8243400-8425-11ea-9ab7-5e663b3cd7b7.png)

To support this, we get to make another "crazy" function called `#IfNotColumnTypeDefault` to support the above call. Will need to add a block of code in the long elseif statement in sql_upgrade_fx.php that will look like this:
https://github.com/openemr/openemr/blob/master/library/sql_upgrade_fx.php#L665-L675

And then you can duplicate the following function with name `columnHasTypeDefault` to have al 4th parameter which carries/checks the default value:
https://github.com/openemr/openemr/blob/master/library/sql_upgrade_fx.php#L70-L78